Gemfile

```diff
@@ -36,6 +36,8 @@ end
 
 group :development, :test do
   gem "byebug"
+  gem "jasmine", "~> 3.6.0"
+  gem "jasmine_selenium_runner", "~> 3.0.0", require: false
```

Comment:
Do we need this for development? or just testing?

Reply:
Hi @theseanything - If you need/want to run the jasmine tests by hand (`bundle exec rake jasmine:ci` or via the browser - `bundle exec rake jasmine`) which can be very useful on occasion, then the `jasmine` gem needs to be in both `development` and `test`.  The `jasmine_selenium_runner` gem can be moved into `test`.

Reply:
Hmmm not sure I understand, because its a test dependency no matter how I run the test. For example "mintiest" is in the :test group, but I can still run tests manually.

Reply:
Hi @theseanything - thanks.  I've moved these into the `test` only section.  This means that it is no longer possible to run `bundle exec rake jasmine:ci` directly.  However.... (see https://github.com/alphagov/smart-answers/pull/4714#discussion_r486211581)